src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/GEOS_LandAssimGridComp.F90

```diff
@@ -145,6 +145,9 @@ subroutine SetServices ( GC, RC )
     type(MAPL_MetaComp), pointer :: MAPL=>null()
     type(ESMF_Config)            :: CF
     character(len=ESMF_MAXSTR)   :: LAND_ASSIM_STR, mwRTM_file
+    character(len=ESMF_MAXSTR)   :: id_string,childname, fmt_str
+    integer                      :: i, ens_id_width, FIRST_ENS_ID, NUM_ENSEMBLE
+    integer, allocatable, dimension(:) :: ens_id, output_incr
```

Comment:
If I'm reading this correctly, there's a variable name "output_incr" and also a subroutine name "output_incr" and, on top of that, the children GCs are labeled OUTPUT_INCR[nnnn]).  
I guess the variable and subroutine are separate things and Fortran sorts out from the context which is which.  
Would  it be better practice to use a different name for, say, the subroutine?  See comment below.

Reply:
I change those names to distinguish them.